assets/src/styles/pages/_site_location.scss

```diff
@@ -7,3 +7,7 @@
  h1, h2 {
     margin: 0;
  }
+
+.waterStatsTable td {
+    text-align: center;
+}
```

Comment:
Looks like this class can now be removed?

Reply:
Good catch. no, I need to rename it with dashes and put it back.

I thought that I did not use any new CSS when I was nuking all the camel-case entries.

Reply:
resolved by replacing it with .water-stats-table